src/components/pages/career-details/career-details-structured-data.tsx

```diff
@@ -20,48 +25,57 @@ const concatDescription = (position: SyPersonioJob): string => {
   );
 };
 
+/**
+ * Create and write structured data based on https://schema.org/JobPosting
+ *
+ * See the Google guide for additional insights:
+ * https://developers.google.com/search/docs/advanced/structured-data/job-posting#structured-data-type-definitions
+ */
 export const CareerDetailsStructuredData = ({
   position,
 }: CareerDetailsStructuredDataProps) => {
-  const employmentType = convertScheduleToEmploymentType(position.schedule);
-
-  const data = {
-    '@context': 'https://schema.org/',
+  const structuredData: WithContext<JobPosting> = {
+    '@context': 'https://schema.org',
     '@type': 'JobPosting',
     title: position.name,
     datePosted: position.createdAt,
     description: concatDescription(position),
     identifier: {
       '@type': 'PropertyValue',
       name: 'Satellytes Digital Consulting GmbH',
-      value: '338504-41230',
+      value: position.jobId,
     },
     hiringOrganization: {
       '@type': 'Organization',
       name: 'Satellytes Digital Consulting GmbH',
       sameAs: 'http://www.satellytes.com',
-      logo: 'https://we-are-hiring.cdn.personio.de/logos/41230/social/6ca8f29950517ac39fcae0d6545a1fe6.png',
     },
-
     jobLocation: {
       '@type': 'Place',
       address: {
         '@type': 'PostalAddress',
-        streetAddress: 'Sendlinger Stra\u00dfe 52',
-        addressLocality: 'M\u00fcnchen',
+        streetAddress: 'Sendlinger Straße 52',
+        addressLocality: 'München',
         addressRegion: 'Bayern',
         postalCode: '80331',
         addressCountry: 'DE',
       },
     },
     jobLocationType: 'TELECOMMUTE',
-    employmentType: employmentType,
+    applicantLocationRequirements: {
+      '@type': 'AdministrativeArea',
+      sameAs: 'http://www.wikidata.org/entity/Q458',
+      name: 'European Union',
```

Comment:
I added this important property for us to highlight that people from EU can work remotely for us. Without this it would fall back to the office location country (Germany). Actually it is Germany right now to keep things simple but I would like to hard code a wider value before we forget about this as it's not configurable in Personio (assumption)

See https://developers.google.com/search/docs/advanced/structured-data/job-posting#remote-jobs for details.
> There are three properties that help Google understand which jobs can be performed from home:

[jobLocationType](https://developers.google.com/search/docs/advanced/structured-data/job-posting#job-location-type): Use this property to specify that the job is a Work from home job.
[applicantLocationRequirements](https://developers.google.com/search/docs/advanced/structured-data/job-posting#applicant-location-requirements): Use this property to specify geographic location(s) in which employees may be located for work from home jobs. At least one country is required.
[jobLocation](https://developers.google.com/search/docs/advanced/structured-data/job-posting#job-location): Use this property to specify where the job can be physically performed. You don't need to use this property if there isn't a physical location where a job can be performed (for example, an office or construction site).


particular data found via https://github.com/schemaorg/schemaorg/issues/2083#issuecomment-510097182

Reply:
Google wants a country or City here. I'm not sure if we can provide EU then.
https://search.google.com/test/rich-results/result/r%2Fjobs?id=SjmH_MBB96uy2ATeGyLIFA

![Screenshot 2022-03-25 at 08 19 33](https://user-images.githubusercontent.com/1701755/160073364-1df5a9f4-b11b-4628-9d87-cce8108efcdb.png)

Reply:
https://support.google.com/webmasters/thread/18006320/job-posting-structured-date-for-remote-jobs-restricted-to-a-geographic-region?hl=en

It looks like we have to add either all EU countries or have to stick to Germany

What would you suggest to do?

Reply:
if we remove it the job location's country will be taken, which is Germany. Let's do that and remove this part. Sorry for the inconvenience.

Reply:
👍